.github/build.yaml.gomplate

```diff
@@ -300,10 +300,8 @@
         with:
           name: build-{{{ $flavor }}}-{{{ $config.arch }}}
           path: build
-      - name: Add build repository
-        run: |
-          export DOCKER_INSTALL=true
-          sudo -E make add_local_repo
+      {{{ tmpl.Exec  "make" "add_local_repo" }}}
+      {{{ tmpl.Exec  "make" "upgrade_deps" }}}
```

Comment:
I am wondering why haven't we done something like that before... feels like I am missing some side effect... Couldn't be that simple solving #1515... 🤔

Reply:
this bites us in the ass down the line. I already did this some time ago, and my thinking was exactly the same LOL

I think it was something to do with cross arch build in which the deps were upgraded with wrong arch packages? Something like that, maybe its now fixed because luet knows better about arches?